server/server.js

```diff
@@ -1,49 +1,68 @@
 const express = require('express');
 const app = express();
 const morgan = require('morgan');
-const pg = require('pg');
+const client = require('./db-client');
 
 app.use(morgan('dev'));
 app.use(express.json());
 
-const Client = pg.Client;
-const dbUrl = 'postgres://localhost:5432/superheroes';
-const client = new Client(dbUrl);
-client.connect();
+// const Client = pg.Client;
+// const dbUrl = 'postgres://localhost:5432/superheroes';
+// const client = new Client(dbUrl);
+// client.connect();
 
-app.get('/api/superheroes', (req, res) => {
+app.get('/api/hero', (req, res) => {
   client.query(`
-    SELECT name, age FROM superheroes;
+    SELECT 
+      hero.id, 
```

Comment:
```suggestion
      id, 
      name,
      age,
      ability
```
If you are only selecting from one table, you can select the columns like above.